src/components/activity/ActivityFormPanel.svelte

```diff
@@ -97,6 +129,12 @@
     });
   }
 
+  $: getActivityMetadataValue = (key: string) => {
```

Comment:
Is there a reason this is a reactive expression and not just a regular function?

Reply:
I may have done this thinking that the metadata object would be somehow bound to the context of the originally declared function? But that's probably not the case? Still trying to understand behaviors here.

Reply:
Aha, so when this is not a reactive function the metadata just references the first selected activity. Is there a better way to implement this?